webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue

```diff
@@ -16,111 +16,41 @@
 -->
 <template>
   <div class="d-flex flex-column white border-radius">
-    <div class="mx-4 my-4">
+    <img
```

Comment:
it would be better to use `<v-img height... max-height... max-width="100%" eager...`